androidApp/src/main/java/com/mbta/tid/mbta_app/android/map/HomeMapView.kt

```diff
@@ -113,25 +110,11 @@ fun HomeMapView(
 
     val selectedVehicle = (state as? State.VehicleSelected)?.vehicle
 
-    val (stop: Stop?, stopFilter: StopDetailsFilter?) =
-        when (state) {
-            is State.StopSelected -> {
-                val state = (state as State.StopSelected)
-                state.stop to state.stopFilter
-            }
-            is State.Overview -> null to null
-
-            is State.VehicleSelected -> {
-                val state = (state as State.VehicleSelected)
-                state.stop to null
-            }
-        }
-
     val showTripRecenterButton =
         when (state) {
             is State.StopSelected,
             is State.Overview -> false
-            is State.VehicleSelected -> !viewportProvider.isVehicleOverview && !isSearchExpanded
```

Comment:
I think this was resolving an old issue related to the trip button appearing over the search results, but that is no longer a problem (and there isn't equivalent logic for the puck recenter button)

Reply:
Oh good call, I'll test that out. It would be easy enough to add to the regular recenter button if that is a problem, I think we just need a check equivalent to `isSearchExpanded && isSearchVisible`

Reply:
Tested it out, and search results + center on a vehicle can't co-exist, so I think this is safe to remove.